tests/coverage/test_gcovr.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+##############################################################
+# Folders structure (28.07.2020)
+# NOTE: Drawing will be removed after approval script
+##############################################################
+# facelift/
+# ├─ build/
+# │  ├─
+# │  └─
+# ├─ ci/
+# │  ├─ appveyor
+# │  └─ travis
+# ├─ cmake/
+# ├─ CMakeLists.txt
+# ├─ src/
+# ├─ tests/
+# │  ├─ asyncfunctions/
+# │  ├─ benchmarking/
+# │  ├─ cmake/
+# │  ├─ CMakeLists.txt
+# │  ├─ combined/
+# │  ├─ coverage/
+# │  │   ├─ read.me
+# │  │   └─ test_gcov.sh*
+
+DIR_CVRG=$(cd "$(dirname "$0")"; pwd -P)
+DIR_TESTS=$(cd "$(dirname "$DIR_CVRG")"; pwd -P)
+DIR_SRC=$(cd "$(dirname "$DIR_TESTS")"; pwd -P)
+BUILD_DIR=$PWD
```

Comment:
be consistent with your variable naming. use "DIR" either as a prefix or as a suffix, but not both.

Reply:
Agree.